contracts/UtilityBrandedToken.sol

```diff
@@ -266,24 +263,12 @@ contract UtilityBrandedToken is EIP20Token, UtilityTokenInterface, Internal {
         internal
         returns (bool success_)
     {
-        require(
-            _beneficiary != address(0),
-            "Beneficiary address should not be zero."
-        );
-
-        require(
-            _amount > 0,
-            "Amount should be greater than zero."
-        );
-
         // Increase the balance of the _account
         balances[_beneficiary] = balances[_beneficiary].add(_amount);
         totalTokenSupply = totalTokenSupply.add(_amount);
 
-        /*
-         * Creation of the new tokens should trigger a Transfer event with
-         * _from as 0x0.
-         */
+        // Creation of the new tokens should trigger a Transfer event with
```

Comment:
I think both commenting styles are accepted as per style guide 🤔 
Does it make sense to change in some places? 
It breaks the consistency in the file, it should just be one style across the file.

Reply:
```
It's recommended to use for code commenting the //.
```

https://github.com/OpenST/mosaic-
contracts/blob/develop/SOLIDITY_STYLE_GUIDE.md#comments

It's recommended ;), we do not push it hard. Subjectively, I like to separate documentation comment style from code comment style. I've changed it intuitively.